### PR TITLE
Fixes #1 | Better formatting for 'between' and Minor code clean-up

### LINKED ISSUE #1: Review or improve the error messages
We need to find a better way to explain to the users what is happening, why it is getting that error, etc. 

I think the current messages are "ugly". Today, we have 26 validators, so we need 26 "beautiful" messages hehe.

Can you help us? Please, enjoy! ;)

### PR DESCRIPTION
**Pull request by mbad0la**
Specs:
* Added check to format `between` case differently.
* Removed redundancy from `package.json`.
* Added `.` for the `between` message.
* Built `boss.min.js` using new additions.

Todo:
* [x] Improve README.md

### CODE REVIEW

**Comment by mbad0la** [comment]
@cezarlz please review this and let me know if I need to change something.

File: src/js/boss.core.js [hunk 1]
```diff
@@ -68,6 +68,18 @@ let Boss = {
               }
 
               if (!validate.call(self, el, rule, rules)) {
+                if (r == 'between') {
+                  let transformation = '';
+
+                  for(let i = 0; i < rule.length ; i += 1) {
```
Comment (cezarlz) [inline src/js/boss.core.js]: I think that we can better the `for` loop, using something like ` for (let i = 0, t = rule.length; i < t; ++i)`

What do you think?

Thanks! :D

Comment (mbad0la) [inline src/js/boss.core.js]: Sure! I'll update the code.

Comment (cezarlz) [inline src/js/boss.core.js]: Cool!

**Comment by cezarlz** [comment]
Did you test it? Can you show me an example of the this implementation? Some screenshot or the final error message.

Thanks!

**Comment by mbad0la** [comment]
@cezarlz here's a screenshot.

newerrormessage

**Comment by cezarlz** [comment]
Amazing @mbad0la. Merging now...

**Comment by mbad0la** [comment]
@cezarlz thanks! 🙂

**Pull request merged**